roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -26,6 +26,7 @@
   environment:
     CEPH_CONTAINER_IMAGE: "{{ ceph_docker_registry + '/' + ceph_docker_image + ':' + ceph_docker_image_tag if containerized_deployment else None }}"
     CEPH_CONTAINER_BINARY: "{{ container_binary }}"
+    CEPH_ROLLING_UPDATE: "{{ 'FALSE' if rolling_update else 'TRUE' }}"
```

Comment:
why not using rolling_update value directly ?

Reply:
because of [this](https://github.com/ceph/ceph-ansible/pull/3730#discussion_r267361433)